core/schema/container.graphqls

```diff
@@ -650,6 +650,13 @@ type Container {
     engine's cache, then it will be compressed using Gzip.
     """
     forcedCompression: ImageLayerCompression
+
+    """
+    Use the specified media types for the published image's layers. Defaults to OCI, which
+    is largely compatible with most recent registries, but Docker may be needed for older
+    registries without OCI support.
+    """
+    mediaTypes: ImageMediaTypes
```

Comment:
nit: I believe expressing defaults here works (we do it for NetworkProtocol), could help make it super obvious

```suggestion
    mediaTypes: ImageMediaTypes = OCIMediaTypes
```

Reply:
Good call! fixed